src/org.spec.ts

```diff
@@ -478,4 +478,60 @@ describe('org', () => {
       expect(error).toHaveProperty('name', 'MongoNotConnectedError');
     });
   });
+
+  describe('deleteOrg', () => {
+    test.each([
+      ['ASCII', ORG_NAME],
+      ['non ASCII', NON_ASCII_ORG_NAME],
+    ])('Existing %s name should remove org', async (_type, name: string) => {
+      await orgModel.create({
+        name: name,
+        memberAccessType: MemberAccessType.OPEN,
+      });
+
+      const methodResponse = await deleteOrg(name, {
+        dbConnection: connection,
+        logger: mockLogging.logger,
+      });
+
+      requireSuccessfulResult(methodResponse);
+      const dbResult = await orgModel.exists({
+        name: name,
+      });
+      expect(dbResult).toBeNull();
+    });
+
+    test('Non existing name should not remove any org', async () => {
```

Comment:
This test makes sure we don't accidentally write something line orgModel.delete({})
Maybe we can come up with a better name